### PR TITLE
Zoidberg changes

### PR DESCRIPTION
**Pull request by dschwoerer**
Some changes to zoidberg.

I also run black (7915e1a ) - if that is not appreciated I can push without that commit ...

### CODE REVIEW

**Comment by bendudson** [comment]
Thanks @dschwoerer ! Black formatting definitely appreciated, and the changes sound good

File: tools/pylib/zoidberg/poloidal_grid.py [hunk 1]
```diff
@@ -349,89 +361,103 @@ def findIndex(self, R, Z, tol=1e-10, show=False):
...
         if show and plotting_available:
-            plt.plot(self.R, self.Z, '.')
-            plt.plot(R, Z, 'x')
-        
+            plt.plot(self.R, self.Z, ".")
+            plt.plot(R, Z, "x")
+
+        cnt = 0
+        underrelax = 1
+        print()
```
Comment (ZedThree) [inline tools/pylib/zoidberg/poloidal_grid.py:402]: Extra `print()` snuck in?

**Review by ZedThree — APPROVED** [review]
LGTM

**Comment by ZedThree** [comment]
One test is timing out, quite annoying. Rather than rerun all the tests (github still won't let you just rerun one!), I'll merge this into 3D metrics branch and let it rerun there

**Pull request merged**